extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -339,9 +340,23 @@ var CreatePage = {
 					}
 				}
 			});
+		} else {
+			// in case create page modal is disabled
+			$( '#WikiaArticle' ).on( 'click', 'a.new', function( e ) {
```

Comment:
Having WW-372 and WW-373 tickets in backlog, is it possible to distinguish between redlinks on special pages and redlingks on other pages?